library/ffau/generator.js

```diff
@@ -82,10 +82,70 @@ htmlGen.finish = function (code) {
     return code;
 };
 
+htmlGen.suffixLines = function (text, suffix) {
+    return text.replace(/(?!\n$)\n/g, suffix + "\n") + suffix;
+};
+
+htmlGen.getAncestors = function (block, ancestors) {
+    if (block.parentBlock_) {
+        ancestors.push(block.parentBlock_);
+        ancestors.push(htmlGen.getAncestors(block.parentBlock_, ancestors));
+    }
+
+    return ancestors.filter(e => e.length === undefined);
+};
+
+// Called with each block/statement to pass onwards custom mapping
 htmlGen.scrub_ = function (block, code) {
+    function appendCommentCode(comment, prefix, suffix) {
```

Comment:
Don't nest functions.

Reply:
Convert this to `const appendCommentCode = (comment, prefix, suffix) => {}` function

Reply:
Still not fixed